modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/EditObjectDetails.tsx

```diff
@@ -222,6 +229,9 @@ export default function EditObjectDetails({
 						values.name
 					)}
 					objectDefinitionId={objectDefinitionId}
+					onExternalReferenceCodeChange={
```

Comment:
If you look at this file, you will see that `setValues` is being passed to most of its child. Follow the pattern, pass `setValues` to `ObjectManagementToolbar` and do something like this: 
```
<ManagementToolbar
  ...
  onExternalReferenceCodeChange={() => setValues({externalReferenceCode})} 
  ...
/>
```